config/rbac/role.yaml

```diff
@@ -154,6 +154,16 @@ rules:
   - seldondeployments
   verbs:
   - '*'
+- apiGroups:
```

Comment:
you are pulling in a commit already in main, can you rebase?

Reply:
Ugh didn't see that, but wait I think rebase took Dharmit's commit into picture hence the additional commit.